plugin/builtin/patchstrategicmergetransformer/go.mod

```diff
@@ -5,11 +5,11 @@ go 1.14
 require (
 	github.com/pkg/errors v0.8.1
 	sigs.k8s.io/kustomize/api v0.5.1
-	sigs.k8s.io/kustomize/kyaml v0.4.2
+	sigs.k8s.io/kustomize/kyaml v0.5.0
 	sigs.k8s.io/yaml v1.2.0
 )
 
 replace (
 	sigs.k8s.io/kustomize/api v0.5.1 => ../../../api
-	sigs.k8s.io/kustomize/kyaml v0.4.2 => ../../../kyaml
+	sigs.k8s.io/kustomize/kyaml v0.5.0 => ../../../kyaml
```

Comment:
I think you can remove this kyaml replace? @monopole

Reply:
I'm leaving those in because for now we need them.
When we release kustomize, we'll remove them - then put them back afterwards :P